superset/databases/api.py

```diff
@@ -265,3 +275,87 @@ def select_star(
             return self.response(404, message="Table not found on the database")
         self.incr_stats("success", self.select_star.__name__)
         return self.response(200, result=result)
+
+    @expose("/schemas/", methods=["GET"])
+    @protect()
+    @safe
+    @statsd_metrics
+    @rison(get_schemas_schema)
+    def schemas(self, **kwargs: Any) -> FlaskResponse:
+        """Get all schemas
+        ---
+        get:
+          parameters:
+          - in: query
+            name: q
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    page_size:
+                      type: integer
+                    page:
+                      type: integer
+                    filter:
+                      type: string
+          responses:
+            200:
+              description: Related column data
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      count:
+                        type: integer
+                      result:
+                        type: object
+                        properties:
+                          value:
+                            type: string
+                          text:
+                            type: string
```

Comment:
Here also, declare a marshmallow schema for the response and reference that. Register your schema on OpenAPI using `openapi_spec_component_schemas` FAB attr